fusequery/query/src/pipelines/transforms/transform_remote_test.rs

```diff
@@ -1,8 +1,8 @@
+// Copyright 2020-2021 The Datafuse Authors.
+//
+// SPDX-License-Identifier: Apache-2.0.
+
 // <<<<<<< HEAD
```

Comment:
Is this an unresolved git conflict?

Reply:
This is under design by @zhang2014  
It will be fixed later

Reply:
Yes. Do you know if this file is unused? If it's unused any longer, let's remove it.

Reply:
It's better to leave a TODO comment on unaddressed issue. :)